docs/developer/release/release_build.py

```diff
@@ -287,6 +287,8 @@ def build_checker_framework_release(
     execute(gradle_cmd, True, False, CHECKER_FRAMEWORK)
     gradle_cmd = "./gradlew assembleForJavac"
     execute(gradle_cmd, True, False, CHECKER_FRAMEWORK)
+    gradle_cmd = "./gradlew getDoLikeJavac"
```

Comment:
This should be part of the `releaseBuild` task rather than called separately here.  (I have the same comment for `assembleForJavac` on the above line.)

Reply:
Could you please make the fix for me?  Thanks!